src/shared/loop-util.c

```diff
@@ -36,6 +36,10 @@
 #include "string-util.h"
 #include "tmpfile-util.h"
 
+#ifndef LOOP_SET_BLOCK_SIZE
+#define LOOP_SET_BLOCK_SIZE 0x4C09
```

Comment:
Please add this to missing_loop.h instead

Reply:
Right... Thanks for pointing it out!